Libraries/AppState/AppState.js

```diff
@@ -152,7 +151,7 @@ class MissingNativeAppStateShim extends EventEmitter {
 // This module depends on the native `RCTAppState` module. If you don't include it,
 // `AppState.isAvailable` will return `false`, and any method calls will throw.
 // We reassign the class variable to keep the autodoc generator happy.
-if (RCTAppState) {
+if (NativeAppState) {
```

Comment:
Btw, since you use `getEnforcing()`, this is always `true`-ish, and we get invariant() instead when it's not available.

This may be a breaking change, so if that's the case we could downgrade it to `get()` instead.

We can try landing this PR then adjust things if it breaks something.

Reply:
oh wow, i didn't realize that. okay. this is more to satisfy flow i guess- so i should check null instead?